ci/tests/vdisk/vdisk_test.py

```diff
@@ -89,3 +85,46 @@ def _get_scrubber_log_size():
                                    loop_device=loop)
 
         assert post_scrubber_logsize > pre_scrubber_logsize, "Scrubber actions where not logged!"
+
+    @staticmethod
+    def ovs_3756_metadata_size_test():
+        """
+        Validate get/set metadata cache size for a vdisk
+        """
+
+        metadata_cache_page_size = 256 * 24
+        default_metadata_cache_size = 8192 * metadata_cache_page_size
+
+        disk_name = 'ovs-3756-disk'
+        loop = 'loop0'
+        vpool = GeneralVPool.get_vpool_by_name(TestVDisk.vpool_name)
+        vdisk = GeneralVDisk.create_volume(size=2, vpool=vpool, name=disk_name, loop_device=loop, wait=True)
+
+        def validate_setting_cache_value(value_to_verify):
+            disk_config_params = GeneralVDisk.get_config_params(vdisk)
+            disk_config_params['metadata_cache_size'] = value_to_verify
+            disk_config_params['write_buffer'] = int(disk_config_params['write_buffer'])
+
+            GeneralVDisk.set_config_params(vdisk, {'new_config_params': disk_config_params})
+            GeneralVDisk.get_config_params(vdisk)
+            actual_value = disk_config_params['metadata_cache_size']
```

Comment:
actual_value will always be equal to the value_to_verify because disk_config_params is not updated by the 2nd get_config_params